core/commands/follow.go

```diff
@@ -0,0 +1,167 @@
+package commands
+
+import (
+	"errors"
+	"io"
+	"strings"
+	"time"
+
+	cmds "github.com/ipfs/go-ipfs/commands"
+	e "github.com/ipfs/go-ipfs/core/commands/e"
+	nc "github.com/ipfs/go-ipfs/namecache"
+
+	"gx/ipfs/QmUyfy4QSr3NXym4etEiRyxBLqqAeKHJuRdi8AACxg63fZ/go-ipfs-cmdkit"
+)
+
+type ipnsFollowResult struct {
+	Result string
+}
+
+var IpnsFollowCmd = &cmds.Command{
+	Helptext: cmdkit.HelpText{
+		Tagline: "Follow IPNS names.",
+		ShortDescription: `
+Periodically resolve and optionally pin IPNS names in the background.
+`,
+	},
+	Subcommands: map[string]*cmds.Command{
+		"add":    ipnsFollowAddCmd,
+		"list":   ipnsFollowListCmd,
+		"cancel": ipnsFollowCancelCmd,
+	},
+}
+
+var ipnsFollowAddCmd = &cmds.Command{
+	Helptext: cmdkit.HelpText{
+		Tagline: "Follow one or more names",
+		ShortDescription: `
+Follows an IPNS name by periodically resolving in the backround.
+`,
+	},
+	Arguments: []cmdkit.Argument{
+		cmdkit.StringArg("name", true, true, "IPNS Name to follow."),
+	},
+	Options: []cmdkit.Option{
+		cmdkit.BoolOption("pin", "Recursively pin the resolved pointer"),
+		cmdkit.StringOption("refresh-interval", "Follow refresh interval."),
```

Comment:
Would be nice to specify/document the default and say what format it's in (like in publish.go/lifetime)

Reply:
added the default, not sure i want to repeat the same documentation for time formats here.